openlibrary/templates/books/edit/web.html

```diff
@@ -24,7 +24,7 @@
                     <label for="link-url">URL</label>
                 </div>
                 <div class="input">
-                    <input type="text" name="url" id="link-url" class="addweb" value="https://"/>
+                    <input type="text" name="url" id="link-url" class="addweb" value="" placeholder="https://..."/>
```

Comment:
These links become relative to Open Library if the `https://` is removed from the value.

I'm going to go ahead and revert this. @rebecca-shoptaw, let me know if that will have any unintended consequences.

Reply:
```suggestion
                    <input type="text" name="url" id="link-url" class="addweb" value="https://" />
```

Reply:
This also makes sense, and should be good! Working on a separate PR to clean up URL validation -- I was actually hoping this would get reverted so I could handle everything over there. Thanks again!